### ISSUE TITLE
Password reset on Openbenchmark.com IMPOSSIBLE!

### ISSUE THREAD

**Issue by gneumann333** [phoronix-test-suite/phoronix-test-suite#422]
Hi,
I forgot my Password for Openbenchmark.com....
So i did the Password reset and got the Email with the Resetlink no prblem...so far

When i click the Link i get directed to as Openbenchmark Page that tells me

Log-In Required
To access this page you must be logged into a valid OpenBenchmarking.org account. Please log-in or create an account.

So how do i reset my Password?!?

**Comment by michaellarabel** [comment]
Yes there are some known issues around password reset and certain passwords not being allowed. Hopefully will get to doing an Openbenchmarking.org update in the next few weeks that is long overdue on my TODO list.
